### PR TITLE
feat: Add role option for warmer

### LINKED ISSUE #46: Way to give different role to warmup lambda function.
Is there a way to give different role for warmup lambda function, so that only that function will have that permission, not others.

### PR DESCRIPTION
**Pull request by juanjoDiaz**
Closes #46 

Full disclosure: I don't have any serverless project setup at the moment so I couldn't test it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,13 +170,14 @@ module.exports.lambdaToWarm = function(event, context, callback) {
 ```yml
 custom:
   warmup:
+    folderName: '_warmup' // Name of the folder created for the generated warmup 
     cleanFolder: false,
```
Comment (chaitanya11) [inline README.md]: Here `cleanFolder: false,` should not contain `,`.

File: README.md [hunk 2]
```diff
@@ -56,7 +56,75 @@ functions:
       - production
       - staging
 ```
-* WarmUP to be able to `invoke` lambdas requires the following Policy Statement in `iamRoleStatements`:
+* WarmUP requires some permissions to be able to `invoke` lambdas.
+
+```yaml
+custom:
+  warmup:
+    cleanFolder: false,
```
Comment (chaitanya11) [inline README.md]: Here `cleanFolder: false`, should not contain `,`.

**Comment by juanjoDiaz** [comment]
Nice catch!
That was there even before my change but it's fixed now :)

**Comment by goncaloneves** [comment]
Thanks @juanjoDiaz for keeping the warmup actually warm 👍 
Thanks @chaitanya11 for request.

Will do my part and merge.

**Pull request merged**